Sources/OpenId4VP/AuthenticationResponse/AuthenticationResponse.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+public struct AuthenticationResponse {
+    let response: [String: String]
+    
+    static func getAuthenticationResponse(_ authorizationRequest: AuthorizationRequest,_ trustedVerifierJSON: [[Verifier]], openId4VpInstance: OpenId4VP) throws -> AuthenticationResponse {
+        
+        Logger.getLogTag(className: String(describing: self))
+        
+        var responseDict: [String: String] = [:]
+        
+        try verifyClientId(verifierList: trustedVerifierJSON, clientId: authorizationRequest.clientId)
+        
+        if (authorizationRequest.presentation_definition != nil) {
+            let presentationDefinition = try PresentationDefinitionValidator.validate(presentatioDefinition:  authorizationRequest.presentation_definition!)
+            
+            responseDict["presentation_definition"] = authorizationRequest.presentation_definition
+            
+            openId4VpInstance.presentationDefinitionId = presentationDefinition.id
```

Comment:
can't we setter instead of updating property directly

Reply:
Added setter methods in openid4vp class.